log.go

```diff
@@ -0,0 +1,16 @@
+package migrator
+
+import "log"
```

Comment:
The default `log` package is not flexible enough and I am not it is a good idea to use it here. Also, it is out of the scope of the MR title

Reply:
The default log is closed. Many times when developing, you need to see the execution of sql and compare it with the error message.

Reply:
I don't think we need this logger for development, especially when this feature is available for package users. 

If you need to check the output of the execution, you can use a debugger or check the output in tests.